Source/Flow/Private/Nodes/Route/FlowNode_NamedRerouteDeclaration.cpp

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "Nodes/Route/FlowNode_NamedRerouteDeclaration.h"
+
+#include "FlowAsset.h"
+#include "Nodes/Route/FlowNode_NamedRerouteUsage.h"
+
+UFlowNode_NamedRerouteDeclaration::UFlowNode_NamedRerouteDeclaration(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+#if WITH_EDITOR
+	Category = TEXT("Named Reroute");
+	NodeStyle = EFlowNodeStyle::Custom;
+#endif
```

Comment:
Consider adding `CustomNodeColor = FLinearColor::MakeRandomColor();` - all unique named reroutes will have random assigned color, instead of default one.